web/packages/core/src/ruffle-player.ts

```diff
@@ -1433,6 +1476,27 @@ export class RufflePlayer extends HTMLElement {
     setIsExtension(isExtension: boolean): void {
         this.isExtension = isExtension;
     }
+
+    private get isPreloaderVisible(): boolean {
+        return (
+            this.container.querySelector<HTMLElement>("#preloader")!.style
+                .display !== "none"
+        );
+    }
+
+    private set isPreloaderVisible(visible: boolean) {
```

Comment:
Personally I prefer old-school `hidePreloader()` and `showPreloader()` over this setter. To avoid name clashing, the existing `showPreloader` field can be renamed to something like `hasPreloader` or `shouldShowPreloader`.